docs/UserGuide.adoc

```diff
@@ -758,47 +830,58 @@ if there are other sites you wish are supported.
 *Q*: I have a problem and this guide did not help. +
 *A*: Post in our https://github.com/CS2103JAN2018-F14-B4/main/issues[issues page] and we will try our best to answer your queries.
 
+*Q*: Can I build upon Bibliotek? +
+*A*: Yes, you are free to build upon Bibliotek. The source code for Bibliotek is available at https://github.com/CS2103JAN2018-F14-B4/main[our github page].
+
 == Command Summary
 
+If all you need is a quick reference on how to use certain commands, you can refer to this list below.
+
 [width="72%",options="header",]
 |=======================================================================
-|Command |Syntax
-|*Add* |`add INDEX`
+|Action |Command syntax
+|*Add book to book shelf* |`add INDEX`
 
 e.g. `add 1`
-|*Add Alias* |`addalias ALIAS_NAME cmd/COMMAND`
+|*Add a new alias* |`addalias ALIAS_NAME cmd/COMMAND`
 
 e.g. `addalias read cmd/edit s/read p/none`
-|*Aliases* |`aliases`
-|*Clear* |`clear`
-|*Decrypt* |`decrypt YOUR_KEY`
-|*Delete* |`delete INDEX`
-|*Delete Alias* |`deletealias ALIAS_NAME`
-|*Edit* |`edit INDEX [s/STATUS] [p/PRIORITY] [r/RATING]`
+|*List existing aliases* |`aliases`
+|*Clear book shelf* |`clear`
+|*Delete book from book shelf* |`delete INDEX`
+|*Delete an existing alias* |`deletealias ALIAS_NAME`
+|*Edit book in book shelf* |`edit INDEX [s/STATUS] [p/PRIORITY] [r/RATING]`
 
 e.g. `edit 2 s/READ r/5`
-|*Encrypt* |`encrypt`
-|*Exit* |`exit`
-|*Help* |`help`
-|*History* |`history`
-|*Library* |`library INDEX`
-|*List* |`list`
-|*Recent* |`recent`
-|*Reviews* |`reviews INDEX`
-|*Search* |`search [SEARCH_TERM] [i/ISBN] [t/TITLE] [a/AUTHOR] [c/CATEGORY]`
+|*Close the App* |`exit`
+|*View help* |`help`
+|*View previous commands* |`history`
+|*Search for book in library* |`library INDEX`
+|*List books in book shelf* |`list [t/TITLE] [a/AUTHOR] [c/CATEGORY] [s/STATUS] [p/PRIORITY] [r/RATING] [by/SORT_BY]`
```

Comment:
Try splitting the command into 2 lines. If that doesn't increase the width of the first column, try modifying the `[width="72%",options="header",]` at the top of the table.
```
|*List books in book shelf* |`list [t/TITLE] [a/AUTHOR] [c/CATEGORY] [s/STATUS]`
`[p/PRIORITY] [r/RATING] [by/SORT_BY]`
```